commands/validations.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"fmt"
+	"strconv"
+)
+
+// contextualAtoi cleans the error output of Atoi calls
```

Comment:
This comment needs to start with a capital letter for GoDoc and golint to be happy.
```suggestion
// ContextualAtoi cleans the error output of Atoi calls
```

Reply:
🤔  i never receive errors like this on my machine. I'll have to look into that. Thank you!